core/main.tf

```diff
@@ -46,3 +47,19 @@ module "secrets_manager" {
     secret_key_base = var.secret_key_base
   }
 }
+
+module "ecs" {
+  source = "../modules/ecs"
+
+  region                        = local.region
+  app_port                      = local.app_port
+  ecr_repo_name                 = local.ecr_repo_name
+  health_check_path             = local.health_check_path
+  subnets                       = module.vpc.private_subnet_ids
+  security_groups               = module.security_group.ecs_security_group_ids
+  alb_target_group_arn          = module.alb.alb_target_group_arn
+  aws_cloudwatch_log_group_name = module.cloudwatch.aws_cloudwatch_log_group_name
+
+  secrets_variables = module.secrets_manager.secrets_variables
+  secret_arns       = module.secrets_manager.secret_arns
```

Comment:
the current setting allows defining the secrets for the containers, what about environment variables? 

When deploying an application, it's likely that we need to define more env variables than the pre-defined ones 
https://github.com/sanG-github/nimble-devops-ic-infrastructure/pull/30/files#diff-2a02e4d133604ac70f36ed7380da6d19545bcbe4d6bd2fb2461c62fb590bafbdR15-R18

e.g. the application environment (staging/ prod), ... how do you plan to support that?

Reply:
fd790e3 ✅